Misc/NEWS.d/next/Build/2021-02-10-17-54-04.bpo-43112.H5Lat6.rst

```diff
@@ -0,0 +1 @@
+Detect musl libc as a separate SOABI.
```

Comment:
Maybe mention the value:

```suggestion
Detect musl libc as a separate SOABI (tagged as ``linux-musl``).
```

(adapt if it’s incorrect to call that part a tag — not sure if SOABI tag is the whole thing including arch)